lib/repository.js

```diff
@@ -641,22 +629,27 @@ Repository.prototype.fetch = function(
  */
 Repository.prototype.fetchAll = function(
```

Comment:
Make sure to remove the comments from the function headers so that the user knows those params are no longer used.